doc/examples/dbdefs/special/__init__.py

```
@@ -16,4 +16,4 @@
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
-from special import *
+from .special import Special
```

Ruff (0.12.2)

`.special.Special` imported but unused; consider removing, adding to `__all__`, or using a redundant alias

(F401)